dollarbot.py

```diff
@@ -1,34 +1,63 @@
 from bs4 import BeautifulSoup
-import urllib3
-import re
 import requests
 import json
-import threading
-
-def dollar_poller():
-    dolar_venta = 0
-    threading.Timer(900.0, dollar_poller).start()
-    http = urllib3.PoolManager()
-    r = http.request('GET', 'http://dolarhoy.bullmarketbrokers.com/')
-    soup = BeautifulSoup(r.data, 'html.parser')
-    soupc = soup.find("h4")
-    soupv = soupc.findNext("h4")
-    dolar_c = re.search(r'\d\d\.\d\d', str(soupc)).group(0)
-    dolar_v = re.search(r'\d\d\.\d\d', str(soupv)).group(0)
-    dif = dolar_venta - float(dolar_v)
-
-    if float(dolar_v) > dolar_venta:
-        post_dollar_in_slack(dolar_c, dolar_v)
-        dolar_venta = dolar_v
-    elif dif > 0.10 or dif < -0.10:
-        post_dollar_in_slack(dolar_c, dolar_v)
-        dolar_venta = dolar_v
-
-def post_dollar_in_slack(compra, venta):
-    jason = {}
-    jason["text"] = "Compra: "+str(compra)+"  Venta: "+str(venta)
-    url = "https://hooks.slack.com/services/"
-    r = requests.post(url, data=json.dumps(jason))
-
-
-dollar_poller()
+import time
+
+config_file = "conf.yml"
+data_file = "data.txt"
+
+# gets prices from DolayHoy and reads data from files, then uses decision logic based on % threshold to notify
+def poll_prices():
+    r = requests.get('http://dolarhoy.bullmarketbrokers.com/')
+    soup = BeautifulSoup(r.content, 'html.parser')
+    soup_b = soup.find("h4")
+    soup_s = soup_b.findNext("h4")
+    buy = get_num(soup_b.getText())
+    sell = get_num(soup_s.getText())
+
+    config = read_conf()
+    slack_hook = config[0]
+    threshold = float(config[1])
+    last_price = float(read_data())
+    difference = sell - last_price
+    percentage = (difference / last_price) * 100.
+    if percentage > threshold:
+        notification = "Buy: ${:04.2f} Sell: ${:04.2f} - Percentage: {:04.1f}% Up".format(buy, sell, percentage)
```

Comment:
The string that you use here could be a module leve constant as well (without the `.format` part)